strimzi/tests/test_unit.py

```diff
@@ -101,3 +120,34 @@ def test_instance_without_operator_endpoint(dd_run_check, check, instance):
         "`cluster_operator_endpoint`, `topic_operator_endpoint` or `user_operator_endpoint`.",
     ):
         dd_run_check(check(instance))
+
+
+@pytest.mark.parametrize(
+    "namespace, instance, endpoint_key",
+    [
+        (
+            "cluster_operator",
+            MOCKED_CLUSTER_OPERATOR_INSTANCE,
+            "cluster_operator_endpoint",
+        ),
+        (
+            "topic_operator",
+            MOCKED_TOPIC_OPERATOR_INSTANCE,
+            "topic_operator_endpoint",
+        ),
+        (
+            "user_operator",
+            MOCKED_USER_OPERATOR_INSTANCE,
+            "user_operator_endpoint",
+        ),
+    ],
+)
+def test_parse_config_populates_only_configured_scrapers(namespace, instance, endpoint_key, check):
+    # Using a fixture simplifies the need to explicity define the class
+    #   This is equivalent to StrimziCheck('strimzi', {}, [instance])
```

Comment:
Not sure this comment is needed because it is used in every other check and is defined right [here](https://github.com/DataDog/integrations-core/blob/4630ba96a42c91af45537764af764c94818cd368/strimzi/tests/conftest.py#L69-L71) but feel free to keep it if you want